src/main/java/io/wispforest/owo/command/debug/DamageCommand.java

```diff
@@ -19,26 +20,24 @@ public class DamageCommand {
 
     public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(literal("damage")
+                .executes(context -> execute(context, DamageSource.OUT_OF_WORLD, context.getSource().getPlayerOrThrow(), 1))
+                .then(argument("source", DamageSourceArgumentType.damageSource())
+                        .executes(context -> execute(context, DamageSourceArgumentType.getDamageSource(context, "source"), context.getSource().getPlayerOrThrow(), 1))
```

Comment:
Again, no randomly dealing one damage. The amount argument should be required

Same goes for the variant with a target entity